CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ cmake_minimum_required(VERSION 3.3 FATAL_ERROR)  # 3.3: EP_add(DEPENDS interface
                                                  # 3.2: continue()
                                                  # 3.1: CMAKE_CXX_STANDARD, BUILD_ALWAYS
 project(psi4
-        LANGUAGES C CXX)
+	LANGUAGES C CXX Fortran)
```

Comment:
Don't add `Fortran` here. Instead add your pkg to https://github.com/psi4/psi4/pull/1401/files#diff-af3b638bc2a3e6c650974192a53c7291R106

Reply:
Oops, I don't remember why I even put this Fortran flag there...
Resolved in a030a01c6.